tests/serac_newmark_test.cpp

```diff
@@ -10,14 +10,14 @@
 #include <memory>
 
 #include "serac/coefficients/coefficient_extensions.hpp"
-#include "../src/serac/integrators/wrapper_integrator.hpp"
+#include "../src/serac/physics/integrators/wrapper_integrator.hpp"
```

Comment:
Not strictly in the scope of this PR but this shouldn't have the `../src/` in it

Reply:
Oops. 
```suggestion
#include "serac/physics/integrators/wrapper_integrator.hpp"
```